docs/browser.md

```diff
@@ -15,12 +15,14 @@ See [Usage](#usage) below for examples.
 
 ### Global
 
-<!-- prettier-ignore -->
```

Comment:
I can understand why here is a ignore now, must be https://github.com/prettier/prettier/pull/6423